sdcm/db_stats.py

```diff
@@ -371,6 +373,11 @@ def get_scylla_versions(self):
                         versions[package.replace('-enterprise', '')] = {'version': match.group(2),
                                                                         'date': match.group(4),
                                                                         'commit_id': match.group(5)}
+                    LOGGER.info("versions[package.replace('-enterprise', '')]=%s", versions[package.replace('-enterprise', '')])
+            if build_id_output:
+                if not 'scylla-server' in versions.keys():
+                    versions['scylla-server'] = {}
+                versions['scylla-server']['build_id'] = build_id_output
```

Comment:
```suggestion
                versions['scylla-server']={'build_id':  build_id_output }
```

Reply:
applied it... it is currently running.
thank you

Reply:
well, reverting it, as `branch-perf-v8` uses an older version of Python, and it overrides the value of the dictionary:
```
'versions': {'scylla-jmx': {'version': '4.7.dev', 'date': '20211130', 'commit_id': 'e3f922c48b'}, 'scylla-server': {'build_id': 'e08696aca14b789f82f8cf3ae25512bfdf43e54c'}, 'scylla-tools': {'version': '4.7.dev', 'date': '20211130', 'commit_id': 'e3f922c48b'}}
```

see the `scylla-server` here